api/client.go

```diff
@@ -317,12 +315,6 @@ func (c *Config) ReadEnvironment() error {
 		if err != nil {
 			return fmt.Errorf("could not parse VAULT_SKIP_VERIFY")
 		}
-	} else if v := os.Getenv(EnvVaultInsecure); v != "" {
```

Comment:
Because both the constants **EnvVaultInsecure** and **EnvVaultSkipVerify** are set to the same value: `VAULT_SKIP_VERIFY`.  So the else if check was testing for the exact same condition as the if check.